code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1401,6 +1401,13 @@
 	Paralyse(paralyse_duration)
 	Jitter(jitter_duration)
 
+/mob/living/carbon/human/proc/asthma_attack()
+	if(disabilities & ASTHMA && !(M_NO_BREATH in mutations))
```

Comment:
You might want to check whether the human has albuterol already in its reagents and in that case not trigger the attack at all.